Feature/fltr 588

werewolfvit opened the pull request.

vsmirnov2077 committed:
FLTR-588 plugin template

vsmirnov2077 committed:
FLTR-588 add auth and captiveSign methods

vsmirnov2077 committed:
FLTR-588 add android code

vsmirnov2077 committed:
FLTR-588 delete constants

akurdyukov commented on CHANGELOG.md:
```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```
Can you write something here pls?

werewolfvit replied:
done

akurdyukov commented on LICENSE:
```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```
Suppose we should use MIT license

werewolfvit replied:
done

akurdyukov commented on example/android/app/local.properties:
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```
this file should be removed, I guess

werewolfvit replied:
done

akurdyukov commented on ios/docusign_flutter.podspec:
```diff
@@ -0,0 +1,24 @@
+                       DESC
+  s.homepage         = 'http://example.com'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Your Company' => 'email@example.com' }
```
I guess we should fill this

werewolfvit replied:
done, need check

vadlit commented on .idea/.gitignore:
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Please add .idea folder in gitignore

werewolfvit replied:
done

vadlit commented on android/.idea/.gitignore:
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
.idea should be in gitignore

werewolfvit replied:
done

vadlit commented on android/src/main/kotlin/com/unitedtraders/mobile/docusign_flutter/DocusignFlutterPlugin.kt:
```diff
@@ -0,0 +1,166 @@
+  }
+
+  private fun captiveSinging(@NonNull call: MethodCall, @NonNull result: Result) {
+    // TODO необходимо решить проблему с запуском вне Activity, т.к. само создание Intent от нас скрыто - это проблема.
```
So signing doesn't work right now?

werewolfvit replied:
fixed

vsmirnov2077 committed:
FLTR-588 savepoint

vsmirnov2077 committed:
FLTR-588 refactoring

vsmirnov2077 committed:
FLTR-588 refactoring

vsmirnov2077 committed:
FLTR-588 MIT license

vsmirnov2077 committed:
FLTR-588 changelog

vsmirnov2077 committed:
FLTR-588 gitignore fix


FLTR-588 fix gitignore


FLTR-588 fix gitignore


FLTR-588 gitignore fix


FLTR-588 podspec fixed

vsmirnov2077 committed:
FLTR-588 fixes

akurdyukov commented on example/README.md:
```diff
@@ -0,0 +1,16 @@
+
+Demonstrates how to use the docusign_flutter plugin.
+
+## Getting Started
```
Pls delete this if it's not really relevant

vsmirnov2077 committed:
FLTR-588 remove README
